README.md

```diff
@@ -80,8 +80,7 @@ this repository and perform an [editable
 install](https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs):
 
 ```sh
-pip install -e ./opentelemetry-api
-pip install -e ./opentelemetry-sdk
+pip install -e ./opentelemetry-sdk/ -e ./opentelemetry-api/ -e ./opentelemetry-semantic-conventions/
```

Comment:
What's the reason behind this change?

Reply:
or you could say `python scripts/eachdist.py develop` and editable install everything 🤷

Reply:
I assume that was a relic from my dev setup I committed on accident, I'll roll that back.